scripts/checkForUpdates.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+VERSION_FILE="https://raw.githubusercontent.com/fuseio/fuse-network/master/Version"
+
+function updateContainers {
+  "$DIR/quickstart.sh"
+}
+
+function getContainerImageNames {
+  docker ps -a | awk '{ print $2 }'
```

Comment:
A minor thing, probably, but I would suggest using:
```bash
docker ps -a --format '{{.Image}}'
```
instead of unreliable $2 field

Reply:
Good spot neater doing it your way :)